packages/widgetbook/lib/src/knobs/builders/knobs_builder.dart

```diff
@@ -142,4 +143,35 @@ class KnobsBuilder {
       ),
     );
   }
+
+  /// Creates a duration input that can be typed in
+  Duration duration({
+    required String label,
+    required Duration initialValue,
+    String? description,
+  }) {
+    return onKnobAdded(
+      DurationKnob(
+        label: label,
+        value: initialValue,
+        description: description,
+      ),
+    )!;
+  }
+
+  /// Creates a duration input that can be adjusted and optionally hold a
+  /// null value
+  Duration? durationOrNull({
+    required String label,
+    Duration? initialValue,
+    String? description,
+  }) {
+    return onKnobAdded(
+      DurationKnob.nullable(
+        label: label,
+        value: initialValue ?? Duration.zero,
```

Comment:
If the `initialValue` is null, it should passed as-is to the Knob.

```suggestion
        value: initialValue,
```

Reply:
True, must have missed this.